cmd/ocp-build-data-enforcer/main.go

```diff
@@ -309,6 +309,7 @@ func (dp *diffProcessor) process() error {
 					"[1]: https://github.com/openshift/ci-tools/tree/master/cmd/ocp-build-data-enforcer",
 					"[2]: https://github.com/openshift/ocp-build-data/tree/openshift-4.6/images",
 				}, "\n"),
+				fmt.Sprintf("Updating %s baseimages to mach ocp-build-data config", d.path),
```

Comment:
This doesn't sound like a valid branch name to me

Reply:
In UpsertPr we used to use the prTitle as the [sourceBranchName](https://github.com/openshift/ci-tools/pull/1237/files#diff-f7d366a8393f769e69b516187d845078L74), so I don't believe I have changed how this tool would behave with this change